### DISCUSSION TITLE
Header Toggle as Visibility option throughout.

### DISCUSSION THREAD

**Discussion by samedarkclouds** [home-assistant/frontend#22026] [Other feature requests]
The new visibility options are awesome. It basically turns every (mostly) card and section into a conditional card. The thing is that we still need to use collapsible/expander-type cards to allow for a tactile way to “open” or "expand ", or if you want it to look like a proper card header you have to make a toggle helper and then add that it its own section or card, or go and add 100 lines in card-mod… or make something a bubble when that isn’t really what you were trying to do. And some of these workarounds actually go the extra step of disabling the ability to use the visibility setting all-together, often adding an extra need for even more workarounds.

So, it makes sense to me that you would have the option in the visibility setting for cards and sections to add and use a “header toggle” right there next to the name/title that just collapses the card/section to the header.

This would greatly reduce the number of hoops to jump through, with not having to build extremely excessive numbers helper-toggles that need to be managed, and numerous stacks of sections under other “toggle sections” and conditional cards.

So, please consider adding a “header toggle” option and a condition to the visibility settings for cards and sections.

Thanks for everything you do!

**Comment by teskanoo** [comment]
I wonder how easy this might be to implement using the new Header cards in 2024.10 - it could just be a tap action option built right into the UI?
